Gemfile

```diff
@@ -7,3 +7,4 @@ gemspec
 # gem 'byebug', group: [:development, :test]
 
 gem 'workarea', github: 'workarea-commerce/workarea'
+gem 'workarea-search_autocomplete', github: 'workarea-commerce/workarea-search-autocomplete', branch: 'improve/SEARCHAUTO-1'
```

Comment:
I will change this once the `improve/SEARCHAUTO-1` branch gets merged, as I need the append point added in the branch.